### ISSUE TITLE
[TASK] - `commands.md`

### ISSUE THREAD

**Issue by code-chaser** [code-chaser/dex#36]
- organize the list of all commands into a table;
- There must be 3 columns:
```
|  #  |    Command    | Description   |
| :-: | :-----------: | :------------ |
```
- change `commands.txt` to `commands.md`

**Comment by CH1NRU5T** [comment]
Hey, is this issue solved? 
If not, can you please assign me this under hacktoberfest?

**Comment by code-chaser** [comment]
Sorry, it's resolved in #37 😅
